src/main/java/com/sublinks/sublinksapi/customemoji/services/CustomEmojiService.java

```diff
@@ -32,12 +32,14 @@ public CustomEmoji updateCustomEmoji(final CustomEmoji customEmoji, final Iterab
 
   private CustomEmoji addKeywords(final CustomEmoji customEmoji, final Iterable<String> keywords) {
     var customEmojiKeywords = new ArrayList<CustomEmojiKeyword>();
-    for (var keyword : keywords) {
-      customEmojiKeywords.add(
-          CustomEmojiKeyword.builder()
-              .keyword(keyword.toLowerCase().trim())
-              .emoji(customEmoji)
-              .build());
+    if (keywords != null) {
```

Comment:
Found a bug here that would throw an NPE if a null list was passed through to the update method. I've updated this to check for null and gracefully return an empty list.
Alternatively if there's a need to always have a list of keywords associated we can throw an exception and fail the update if the iterator is null